R/financial_functions.R

```diff
@@ -0,0 +1,28 @@
+#' Calculates annual net profits on the company-technology level for the
+#' baseline and late and sudden scenarios. Climate laggards which need to build
+#' out their production in increasing technologies to compensate for their
+#' missed targets, are "punished" by adjusting the net profit margin on their
+#' additional build out based on their proximity to target and market share within
+#' the given technology. This ensures that late build out will not
```

Comment:
I would argue it would be good to provided a more concrete information how the punishment happens.